edm4hep.yaml

```
@@ -398,7 +398,7 @@ datatypes:
       - float                energy [GeV]        // energy of the cluster
       - float                energyError [GeV]   // error on the energy
       - edm4hep::Vector3f    position [mm]       // position of the cluster
-      - edm4hep::CovMatrix3f positionError   // covariance matrix of the position
+      - edm4hep::CovMatrix3f positionError [mm**2]  // covariance matrix of the position
```

There is one prior instance of the use of ^ to indicate an exponent in the yaml file.
```Chi^2```

```suggestion
      - edm4hep::CovMatrix3f positionError [mm^2]  // covariance matrix of the position
```

---

There is also one prior instance of using `**` for units

https://github.com/key4hep/EDM4hep/blob/1cda5c55016eb760057607f7dba7dd591b859f59/edm4hep.yaml#L404

I don't have strong opinion whether to use `**` or `^` for units. But I think for `Chi^2` I'd prefer to rename it to "chi-square" so it's a plain text comment?

---

I didn't want to be annoying but I agree with Andre on this one, `^2` I feel looks more physics-like and `**2` more computer-like, so I prefer ^2. At the same time, I think there was also a previous instance of **2 somewhere in the yaml file.

---

No problem, you are never annoying :smile: `chi^2` is now `chi-squared`, units use `^` instead of `**`

---

Thanks for indulging me!

---

Yaml parser doesn't like `^`?

---

podio validation is at fault here:
https://github.com/AIDASoft/podio/blob/7a3598b187a014b764c3467c251b045057cf91f4/python/podio_gen/podio_config_reader.py#L39-L41

---

Then we either revert to `**` or this waits for https://github.com/AIDASoft/podio/pull/755 (and becomes another reason to bump in #415)

---

Go for `**`, I don't think my whims are worth bumping the required podio version for.